js/app.js

```diff
@@ -0,0 +1,83 @@
+
+//question 1
+
+let obj ={
+	insideObj: {
+		key1: 'yo';
+		key2: 12
+	}
+
+}
+
+console.log(insideObj.key1())
+
+// question 2
+
+let obj = {
+	 {
+		someObject: {
+			someProperty: 'hello', 'hi', 'how are you'
+ 		}
+	}
+ }
+
+ // question 3
+
+ let obj = {
+ 	property () => { ['dare', 'james']
```

Comment:
missing colon:
```javascript
property: ()=>{
   ['dare', 'james']
}
```
also your function doesn't do anything!  it just has an array by itself that isn't set to a variable or printed or anyhing....